src/pages/claim-details/claim-actions.tsx

```diff
@@ -0,0 +1,141 @@
+          </div>
+          <div className={styles.actionPanel}>
+            <Button type="primary" disabled={disableActions} onClick={handleAcceptCounter}>
+              Accept Counter
+            </Button>
+            <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
+              Escalate to Kleros
+            </Button>
+          </div>
+        </div>
+      );
+
+    case 'Accepted':
+      return (
+        <div className={styles.actionSection}>
+          <p>{abbrStr(claim.claimant)}</p>
+          <div className={styles.actionMainInfo}>
+            Accepted <br />
+            counter of <br />
+            {formatApi3(claim.counterOfferAmount!)} API3
+          </div>
+        </div>
+      );
+
+    case 'Appealed':
+      return (
+        <div className={styles.actionSection}>
+          <p>{abbrStr(claim.claimant)}</p>
+          {claim.counterOfferAmount ? (
+            <div className={styles.actionMainInfo}>
+              Appealed counter of <br />
+              {formatApi3(claim.counterOfferAmount!)} API3 <br />
+              to Kleros
+            </div>
+          ) : (
+            <div className={styles.actionMainInfo}>Appealed to Kleros</div>
+          )}
+        </div>
+      );
+
+    case 'Rejected':
+      if (claim.statusUpdatedBy === 'arbitrator') {
+        return (
+          <div className={styles.actionSection}>
+            <p>Kleros</p>
+            <div className={styles.actionMainInfo}>Rejected</div>
+            <div className={styles.actionPanel}>
+              <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
+                Appeal
+              </Button>
+            </div>
+          </div>
+        );
+      }
+
+      return (
+        <div className={styles.actionSection}>
+          <p>API3 Multi-sig</p>
+          <div className={styles.actionMainInfo}>Rejected</div>
+          <div className={styles.actionPanel}>
+            <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
+              Escalate to Kleros
+            </Button>
+          </div>
+        </div>
+      );
+
+    case 'Resolved':
+      if (claim.statusUpdatedBy === 'arbitrator') {
+        return (
+          <div className={styles.actionSection}>
+            <p>Kleros</p>
+            <div className={styles.actionMainInfo}>
+              Approved <br />
+              counter of <br />
+              {formatApi3(claim.resolvedAmount!)} API3
+            </div>
+            <div className={styles.actionPanel}>
+              <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
+                Appeal
+              </Button>
+            </div>
+          </div>
+        );
+      }
+
+      return (
+        <div className={styles.actionSection}>
+          <p>API3 Multi-sig</p>
+          <div className={styles.actionMainInfo}>Approved</div>
+        </div>
+      );
+
+    default:
```

Comment:
If you remove this TS will be able to warn you in case there will be new status/ you forget to handle one.